### PR TITLE
User importer

### PR DESCRIPTION
**Pull request by JuanVT**
Adding URL, view, temporary template and setting up 'import_export' module to create the user importer

### CODE REVIEW

File: control/users/urls.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 from django.conf.urls import url
 
-from control.users.views import get_users, get_user_details
+from control.users.views import get_users, get_user_details, users_import
 
 urlpatterns =[
```
Comment (TysonRV) [inline control/users/urls.py]: Unnecessary line break

File: control/users/urls.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 
     url(r'^users/$', get_users, name='get_users'),
     url(r'^user/(?P<user_id>\d+)/$', get_user_details, name='get_user_details'),
+    url(r'^import-user/$', users_import, name='users_import'),
```
Comment (TysonRV) [inline control/users/urls.py]: Unnecessary line break

File: control/users/views.py [hunk 3]
```diff
@@ -40,6 +43,30 @@ def get_user_details(request, user_id):
...
+
+        user_resource = UserResource()
+        data_set = Dataset()
+        new_users = request.FILES['myfile']
+
+        import_data = data_set.load(new_users.read(), format='csv')
+        result = user_resource.import_data(data_set, dry_run=True)
+
+        if not result.has_errors():
+
```
Comment (TysonRV) [inline control/users/views.py:60]: Unnecessary line break

File: control/users/views.py [hunk 4]
```diff
@@ -40,6 +43,30 @@ def get_user_details(request, user_id):
...
 
+def users_import(request):
+
+    context = {}
+
+    if request.method == 'POST':
+
+        user_resource = UserResource()
+        data_set = Dataset()
+        new_users = request.FILES['myfile']
```
Comment (TysonRV) [inline control/users/views.py:54]: Look for a better name for the file object

File: settings.py [hunk 5]
```diff
@@ -39,6 +39,7 @@
     'django.contrib.staticfiles',
 
     'control.users',
+    'import_export',
```
Comment (TysonRV) [inline settings.py:42]: Avoid mixing your own apps with external ones.

```
INSTALLED_APPS = {
    'django.app1',
    'django.app2',

    'external.app1',
    'import_export',

    'control.users',
}
```

File: control/users/views.py [hunk 6]
```diff
@@ -40,6 +43,30 @@ def get_user_details(request, user_id):
     return HttpResponse(template.render(context=context, request=request))
 
 
+def users_import(request):
+
+    context = {}
+
+    if request.method == 'POST':
+
```
Comment (TysonRV) [inline control/users/views.py:51]: This line is also unnecessary, indent does the job

File: control/users/views.py [hunk 7]
```diff
@@ -40,6 +43,30 @@ def get_user_details(request, user_id):
...
+        import_data = data_set.load(new_users.read(), format='csv')
+        result = user_resource.import_data(data_set, dry_run=True)
+
+        if not result.has_errors():
+
+            user_resource.import_data(data_set, dry_run=False)
+
+        else:
+
```
Comment (TysonRV) [inline control/users/views.py:64]: Ditto unnecessary line

File: control/users/views.py (hunk 4 above)
Comment (TysonRV) [reply control/users/views.py:54]: Not sure where you can change it in the form tho... It might be trickier than I thought

**Comment by JuanVT** [comment]
Thanks for checking this, I'll apply the changes in the next commit

**Pull request merged**